notebooks/07_Experiment_Tracking/07_01_Metadata.ipynb

```diff
@@ -45,28 +45,41 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
+    "scrolled": true
+   },
    "outputs": [],
    "source": [
-    "# To use the latest publish `kfmd` library, you can run:\n",
-    "!pip install kfmd --user\n",
+    "# To use the latest publish `kubeflow-metadata` library, you can run:\n",
+    "!pip install kubeflow-metadata --user\n",
     "\n",
-    "# Install other packages used in the turorial:\n",
-    "!pip install pandas --user"
+    "# Install other packages:\n",
+    "!pip install pandas --user\n",
+    "# Then restart the Notebook kernel."
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 1,
```

Comment:
Let's clean up output before the PR.

Reply:
Okay, I thought we need the output for presentation. After check previous notebook, we don't need. Will fix it now.